src/components/updatemenus/draw.js

```diff
@@ -315,8 +337,32 @@ function drawButtons(gd, gHeader, gButton, menuOpts) {
 
     buttons.call(styleButtons, menuOpts);
 
-    // translate button group
-    Lib.setTranslate(gButton, menuOpts.lx, menuOpts.ly);
+    scrollBox.enable();
+
+    var active = menuOpts.active,
+        i;
+    if(isVertical) {
+        if(scrollBox._vbar) {
+            var translateY = 0;
+            for(i = 0; i < active; i++) {
+                translateY += menuOpts.heights[i] + constants.gapButton;
```

Comment:
Oh! The [position of the scrollbox](https://github.com/n-riesco/plotly.js/blob/13508daf68fcb77499d2b3975448675386753af3/src/components/updatemenus/draw.js#L302) is wrong! This happens in the `up` direction, when the scrollbox doesn't fit within the chart height. Something similar may happen in the `left` direction.